example/particles/particles.cpp

```diff
@@ -577,40 +506,161 @@ TaskCollection ParticleDriver::MakeParticlesCreationTaskCollection() const {
   return tc;
 }
 
-TaskCollection ParticleDriver::MakeParticlesUpdateTaskCollection() const {
+// TODO(BRR) To be used in the future, currently there appears to be a bug in iterative
+// tasking
```

Comment:
Might be worth adding an issue so we don't lose track of this.

Reply:
Yes you're right -- written up in #1155